sintr_common/lib/tasks.dart

```diff
@@ -253,7 +285,7 @@ class TaskController {
 
     // TODO: Add co-ordination of the job to outside the control scripts
     var query = _db.query(_TaskModel)
-      ..filter("lifecycleState =", READY_STATE);
+      ..filter("lifecycleState =", READY_STATE)..limit(10);
```

Comment:
What does limit(10) do? Limit number of total results returned? If yes, then a worker will attempt to return the next unallocated task in a group of 10 and give up (return null) if unallocated task not found. I think that makes sense.